demos/src/footer.mustache

```diff
@@ -21,11 +21,12 @@
 					</div>
 				</div>
 				{{/matrix}}
+				<div class="o-footer__matrix-group o-footer__matrix-group--1">
+					<h3 class="o-footer__matrix-title last__static-link" aria-controls="o-footer-section-5" aria-hidden="false">
```

Comment:
The classname `last__static-link` should be a modifier class on `o-footer__matrix-title` such as `o-footer__matrix-title--external-link`.

Reply:
Removal of the `aria-controls` should prevent this item being collapsed on smaller screen sizes. Removing the need for the `:not` in the scss